awx/main/tasks.py

```diff
@@ -1482,38 +1479,85 @@ def run(self, pk, **kwargs):
                     del params[v]
 
             self.dispatcher = CallbackQueueDispatcher()
-            if self.instance.is_isolated() or containerized:
-                module_args = None
-                if 'module_args' in params:
-                    # if it's adhoc, copy the module args
-                    module_args = ansible_runner.utils.args2cmdline(
-                        params.get('module_args'),
-                    )
-                # TODO on merge: delete if https://github.com/ansible/awx/pull/8185 is merged
-                if not os.path.exists(os.path.join(private_data_dir, 'inventory')):
-                    shutil.move(
-                        params.pop('inventory'),
-                        os.path.join(private_data_dir, 'inventory')
-                    )
 
-                ansible_runner.utils.dump_artifacts(params)
-                isolated_manager_instance = isolated_manager.IsolatedManager(
-                    self.event_handler,
-                    canceled_callback=lambda: self.update_model(self.instance.pk).cancel_flag,
-                    check_callback=self.check_handler,
-                    pod_manager=pod_manager
-                )
-                status, rc = isolated_manager_instance.run(self.instance,
-                                                           private_data_dir,
-                                                           params.get('playbook'),
-                                                           params.get('module'),
-                                                           module_args,
-                                                           ident=str(self.instance.pk))
-                self.finished_callback(None)
+            if not isinstance(self.instance, ProjectUpdate):
```

Comment:
This block is a little weird since they currently both appear to do the same thing.  Maybe it would be better to just leave an `if not isinstance(self.instance, ProjectUpdate):` statement w/ a code comment inside of it for now?

Reply:
I'm not quite ready for this level of feedback yet, hence the Draft status. This conditional is just a hack that I plan on removing.

Reply:
Okey-dokey.